modules/installation-guide/partials/assembly_mounting-secrets-or-a-configmap-as-file-or-env-variable-in-container.adoc

```diff
@@ -0,0 +1,28 @@
+:parent-context-of-mounting-a-secret-or-a-configmap-as-a-file-or-an-environment-variable-into-a-container: {context}
+
+[id="mounting-a-secret-or-a-configmap-as-a-file-or-an-environment-variable-into-a-container_{context}"]
+= Mounting a Secret or a ConfigMap as a file or an environment variable into a {prod-short} container
+
+:context: mounting-a-secret-or-a-configmap-as-a-file-or-an-environment-variable-into-a-eclipse-che-container
+
+Secrets are {platforms-name} objects that store sensitive data such as:
+
+* usernames
+* passwords
+* authentication tokens
+* and configurations
+
+in an encrypted form.
+
+Users can mount a {platforms-name} Secret that contains sensitive data or a ConfigMap in a {prod-short} container as:
+
+* a file
+* an environment variable
+
+The mounting process uses the standard {platforms-name} mounting mechanism, but it requires additional annotations and labeling.
```

Comment:
Maybe it makes sense to note that after configmap/secret is updated, it's needed to rollout the target deployment, until the corresponding issue is not fixed (not sure if it's even created)